template/flink-distribution/conf/flink-conf.yaml

```diff
@@ -53,3 +53,11 @@ taskmanager.state.local.root-dirs: file:///local/state/recovery
 
 jobmanager.memory.process.size: 1g
 taskmanager.memory.process.size: 4g
+
+#==============================================================================
+# Support easy upgrades as the module.yaml file updates
+#==============================================================================
+
+pipeline.auto-generate-uids: false
+execution.savepoint.eignore-unclaimed-state: true
```

Comment:
did u mean `execution.savepoint.ignore-unclaimed-state` ?

Reply:
other than that lgtm

Reply:
I have a question about enforcing `execution.savepoint.eignore-unclaimed-state` config - is there a possibility that an operator created in a StateFun application may not be checkpointed? If that is not the case, then I have a use case which enforcing this config might hurt a little. I am using the state bootstrapping functionality. In case I miss to bootstrap a state, it is going to be an empty state without me noticing I actually forgot to fill that state. What do you think?

Reply:
no, these uid's have nothing to do with user code functions.